pubtools/_pulp/tasks/set_maintenance/set_maintenance_off.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import re
+
+from .base import SetMaintenance
+
+
+LOG = logging.getLogger("set-maintenance")
+
+
+class SetMaintenanceOff(SetMaintenance):
+    """Unset repositories maintenance mode.
+
+    See "pub maintenance-on --help" for more information on maintenance mode.
```

Comment:
I would rather drop the specific mention of Pub.  It should be possible to understand what this command is doing without having to refer back to Pub.